### DISCUSSION TITLE
为什么集群需要 Overlay 网络 - 面向信仰编程 · /whys-the-design-overlay-network

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#615] [Announcements]
https://draveness.me/whys-the-design-overlay-network/

**Comment by 0x2cc** [comment]
有点好奇你每天哪来这么多时间写文章哈哈哈，写一篇分享下这个话题吧

**Comment by draveness** [comment]
> 有点好奇你每天哪来这么多时间写文章哈哈哈，写一篇分享下这个话题吧

https://draveness.me/few-words-time-management/

**Comment by rfyiamcool** [comment]
哈哈，你不怕面试的时候被问 "你写博客这么频繁，是不是工作不饱和？" 。。。。 因为我被好几家公司问过这个话题，然后就把更新的频率放低。。。。😅

**Comment by draveness** [comment]
> 哈哈，你不怕面试的时候被问 "你写博客这么频繁，是不是工作不饱和？" 。。。。 因为我被好几家公司问过这个话题，然后就把更新的频率放低。。。。😅

不怕也没被问过，能问出来的就不要考虑了

**Comment by twoconk** [comment]
写的不错，老乡在西安？

**Comment by draveness** [comment]
> 写的不错，老乡在西安？

不在

**Comment by ohmyadd** [comment]
再补充一个协议 Geneve

**Comment by koktlzz** [comment]
> 当某个容器向集群中发送 ARP 请求，集群中的全部容器都会收到 ARP 请求，这时会带来极高的网络负载。

如果是对目标容器的IP发起arp请求的话，节点应该是不会回答的吧，为什么其他节点中的容器也会收到呢？

**Comment by panjianning** [comment]
挺好挺好

**Comment by MUCZ** [comment]
不同node的pod不在同一子网，这里的图全是同一子网的，请问为什么？

**Comment by lizhen-xx** [comment]
通透~
